commands/about.js

```diff
@@ -0,0 +1,30 @@
+// console logging
+const namespace = 'Slash';
+const { logger } = require('../helpers/logger.js');
+
+// requires
+const { discord: { channelIds: { gettingStartedId }, userIds } } = require('../../tokens.json');
+const { version } = require('../package.json');
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const { simpleEmbed, linkRow } = require('../helpers/embedder.js');
+
+module.exports = {
+    data: new SlashCommandBuilder()
+        .setName('about')
+        .setDescription('Information about this project.'),
+    async execute(interaction) {
+        logger(namespace, `About - Initiated by "${(interaction.user != undefined ? interaction.user.username : 'Unknown')}"`, 'Sent', new Date());

```

Comment:
`(interaction.user != undefined ? interaction.user.username : 'Unknown')`
You have this specific code repeated in at least a dozen places. Factor it out into a global utility method, something like `displayUsername(interaction.user)` or `usernameStr(interaction.user)`.